changelog.d/21194_dot_graph_attributes.feature.md

```diff
@@ -0,0 +1,3 @@
+Adds support for additional graph configuration on each component, for adding arbitrary graphviz node attributes when generating a graph via `vector graph`.
```

Comment:
```suggestion
Adds support for additional graph configuration on each component so that users can add arbitrary graphviz node attributes when generating a graph via `vector graph`.
```